<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommandCaptureWidget</class>
 <widget class="QWidget" name="CommandCaptureWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="query_only_checkbox">
         <property name="text">
          <string>Show query-only cmds [QO]</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="set_only_checkbox">
         <property name="text">
          <string>Show set-only cmds [SO]</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="excluded_checkbox">
         <property name="text">
          <string>Show excluded cmds [EX]</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="method_checkbox">
         <property name="text">
          <string>Show methods [M]</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="raw_command_checkbox">
         <property name="text">
          <string>Show cmd &lt;CMD&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="capture_button">
         <property name="text">
          <string>Capture</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="expand_button">
         <property name="text">
          <string>Expand all</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="collapse_button">
         <property name="text">
          <string>Collapse all</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="tree_view"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
